packages/itwinui-css/src/button/button.scss

```
@@ -33,6 +33,7 @@
 }
 
 @mixin iui-button-icon {
+  &,
```

Adding display prop directly works better. Inherit or inline-flex.

```suggestion
  display: inherit;
```

---

used `inline-flex` in [`e209c85`](https://github.com/iTwin/iTwinUI/pull/854/commits/e209c85691cde4bd1babd3ba88ceb125d562361c).

found test was failing in skip-to-content because class was applied on wrong element.  fixed in [`9dbd5e1`](https://github.com/iTwin/iTwinUI/pull/854/commits/9dbd5e196c6c92cdb96d52ddde57680ca7c259dd), all tests should pass now